tests/Unit.Tests/Controllers/FilterControllerTests.cs

```diff
@@ -146,5 +148,32 @@ public void QualificationGetRedirectsToNewApp()
             _redirectUrlMock.Verify(x => x.RedirectToNewApp(), Times.AtLeastOnce);
             result.Url.Should().Be(actualRedirectPath);
         }
+
+        [Test]
+        public void SubjectWizardGetHttpGetTest()
+        {
+            var result = _filterController.SubjectWizardGet(_resultsFilter);
+            result.Should().BeOfType<ViewResult>();
+            var viewResult = result as ViewResult;
+            viewResult?.Model.Should().BeOfType(typeof(SubjectFilterViewModel));
+            _mockApi.Verify(x => x.GetSubjectAreas(), Times.Exactly(1));
+            viewResult.ViewData["IsInWizard"].Should().Be(true);
+        }
+
+        [Test]
+        public void SubjectWizardGetRedirectsToNewApp()
+        {
+            _mockFlag.Setup(x => x.RedirectToRails).Returns(true);
+            string actualRedirectPath = "start/subject?query";
+
+            var redirectObject = new RedirectResult(actualRedirectPath);
+            _redirectUrlMock.Setup(x => x.RedirectToNewApp())
+                .Returns(redirectObject);
+
+            var result = _filterController.SubjectWizardGet(_resultsFilter) as RedirectResult;
+            result.Should().Be(redirectObject);
+            _redirectUrlMock.Verify(x => x.RedirectToNewApp(), Times.AtLeastOnce);
```

Comment:
can you be more precise than `AtLeastOnce`

Reply:
fixed